EIPS/eip-7702.md

```diff
@@ -44,6 +44,15 @@ rlp([chain_id, nonce, max_priority_fee_per_gas, max_fee_per_gas, gas_limit, dest
 authorization_list = [[chain_id, address, nonce, y_parity, r, s], ...]
 ```
 
+Transaction is considered invalid if authorization list items can't be decoded as:
+
+* `chain_id`: unsigned 256-bit integer.
+* `nonce`: unsigned 64-bit integer.
+* `address`: 20 bytes array.
+* `y_parity`: Value 0 or 1.
+* `r`: 32 bytes array.
```

Comment:
I believe this should also be an unsigned 256-bit integer for RLP purposes?

Reply:
Makes sense

Reply:
@lightclient it is updated